packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -0,0 +1,130 @@
+        const container = await loader.resolve({ url: documentLoadUrl });
+        opc.addDeltaManagers(container.deltaManager);
+        return container;
+    }
+
+    beforeEach(async () => {
+        deltaConnectionServer = LocalDeltaConnectionServer.create();
+        opc = new OpProcessingController();
+
+        // Create a Container for the first client.
+        const container = await createContainer();
+        const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+
+        assert.strictEqual(container.deltaManager.active, false, "active");
+        assert.strictEqual(container.deltaManager.readonly, false, "readonly");
+
+        assert.strictEqual(dataObject.runtime.existing, false, "existing");
+        assert.strictEqual(dataObject.runtime.connected, true, "connected");
+        assert.notStrictEqual(dataObject.runtime.clientId, undefined, "clientId");
+
+        dataObject.root.set("test","key");
+        await opc.process();
+    });
+
+    it("Validate Properties on Loaded Container With No Delta Stream", async () => {
+        // Load the Container that was created by the first client.
+        const container = await loadContainer(true) as Container;
+
+        assert.strictEqual(container.connected, true, "container.connected");
+        assert.strictEqual(container.clientId, undefined, "container.clientId");
+        assert.strictEqual(container.existing, true, "container.existing");
+        assert.strictEqual(container.readonly, true, "container.readonly");
+        assert.strictEqual(container.readonlyPermissions, true, "container.readonlyPermissions");
+
+        const deltaManager = container.deltaManager as DeltaManager;
+        assert.strictEqual(deltaManager.active, false, "deltaManager.active");
+        assert.strictEqual(deltaManager.readonly, true, "deltaManager.readonly");
+        assert.strictEqual(deltaManager.readonlyPermissions, true, "deltaManager.readonlyPermissions");
+        assert.strictEqual(deltaManager.connectionMode, "read", "deltaManager.connectionMode");
+        assert.strictEqual(deltaManager.storageOnly, true, "deltaManager.storageOnly");
+
+        const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+        assert.strictEqual(dataObject.runtime.existing, true, "dataObject.runtime.existing");
+        assert.strictEqual(dataObject.runtime.connected, true, "dataObject.runtime.connected");
+        assert.strictEqual(dataObject.runtime.clientId, undefined, "dataObject.runtime.clientId");
+
+        assert.strictEqual(dataObject.root.get("test"), "key", "mapKey");
+
+        container.close();
+    });
+
+    it("can't send ops", async () => {
+        const container = await loadContainer(true) as Container;
+        const dataObject = await requestFluidObject<ITestFluidObject>(container, "default");
+
+        let err = false;
+        try {
+            dataObject.root.set("asdfasdf", "asfdasdfasdfasdf");
```

Comment:
>dataObject [](start = 12, length = 10)

a pattern i like to do is to add an assert.fail() in the try, then you don't need to err flag